vmware_aria_operations_integration_sdk/project.py

```diff
@@ -107,27 +109,61 @@ def name(self) -> str:
         return get_project_name(self.path)
 
     def record(self) -> None:
-        config_file = os.path.join(self.path, "config.json")
+        config_file = os.path.join(self.path, CONFIG_FILE_NAME)
+        connections_file = os.path.join(self.path, CONNECTIONS_FILE_NAME)
         set_config_value(
-            "connections", [conn.__dict__ for conn in self.connections], config_file
+            "connections",
+            [conn.__dict__ for conn in self.connections],
+            connections_file,
         )
         set_config_value("docker_port", self.docker_port, config_file)
 
     @classmethod
     def extract(cls, path: str) -> Project:
-        local_config_file = os.path.join(path, "config.json")
+        local_config_file = os.path.join(path, CONFIG_FILE_NAME)
+        connections_file = os.path.join(path, CONNECTIONS_FILE_NAME)
+
+        if not os.path.isfile(connections_file):
+            if os.path.isfile(
+                local_config_file
+            ):  # if there is a config.json, but not a connections.json file, is a sign that is an old version of the config.json file
+                logger.warning(
+                    f"Possible {CONFIG_FILE_NAME} file detected. To learn more see https://vmware.github.io/vmware-aria-operations-integration-sdk/troubleshooting_and_faq/collections/#how-can-i-migrate-my-old-configjson-file-deprecated-fields-detected-in-configjson-file-warning"
```

Comment:
I believe this will display as:
"Possible config.json file detected. [...]" Which I don't think describes the situation.

Also, not having a "connections.json" file doesn't strike me as necessarily something we should warn about. For example, someone has cloned a repo someone else made and is running it for the first time. They won't have a 'connections.json' file, because we're excluding it from source control, but that doesn't mean there's a problem, just that they haven't created one yet.

I think the warning only makes sense if there's also a "connections" key inside the 'config.json' file. (and even then, it may be more appropriate as just an info log)